projects/boringssl/Dockerfile

```
@@ -15,7 +15,10 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install -y cmake ninja-build golang
-
+RUN apt-get update && apt-get install -y cmake ninja-build golang curl wget \
+    binutils cmake ninja-build liblzma-dev libz-dev pkg-config autoconf libtool
```

You have `cmake` and `ninja-build` twice here.

```suggestion
    binutils liblzma-dev libz-dev pkg-config autoconf libtool
```